cpp-linter/src/clang_tools/clang_tidy.rs

```diff
@@ -258,19 +258,15 @@
     if let Some(db) = &clang_params.database {
         cmd.args(["-p", &db.to_string_lossy()]);
     }
-    if let Some(extras) = &clang_params.extra_args {
-        for arg in extras {
-            cmd.args(["--extra-arg", format!("\"{}\"", arg).as_str()]);
-        }
+    for arg in &clang_params.extra_args {
+        cmd.args(["--extra-arg", format!("\"{}\"", arg).as_str()]);
```

Comment:
Removing the quotes will cause the arguments to be interpreted incorrectly because `arg.as_str()` may be interpreted by the shell as another option instead of `--extra-arg`'s value.